### PR TITLE
feat: fallback to development stage when Rails is not fully initialized

### PR DESCRIPTION
**Pull request by thomasnemer**
This PR intends to fallback to development stage when Rails is used but not fully initialized.
(`Rails` defined but not `::Rails.env` yet)

### CODE REVIEW

**Review by Djiit — APPROVED** [review]

File: lib/stagery.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ module Stagery
   def self.stage
     @_stage ||=
       ActiveSupport::StringInquirer.new(
-        ENV["STAGERY_STAGE"] || (defined?(Rails) ? ::Rails.env : "development")
+        ENV["STAGERY_STAGE"] || (defined?(Rails.env) ? ::Rails.env : "development")
```
Comment (ClairePhi) [inline lib/stagery.rb:11]: I do not know well this gem. Is there a need to update the README?

Comment (thomasnemer) [reply lib/stagery.rb:11]: I think this PR only fixes an edge case of an already documented feature (fallback to `Rails.env` if `STAGERY_STAGE` isn't set).

**Comment by Djiit** [comment]
@beauraF Can we have a review please ?

**Comment by beauraF** [comment]
Migrated inside doctolib codebase.

**Pull request closed without merging**